hash/zx81_cass.xml

```diff
@@ -5833,13 +6010,28 @@ Known dumps not yet added (as of 2023-06-14):
 		<description>Sabotage</description>
 		<year>1982</year>
 		<publisher>Sinclair Research</publisher>
+		<info name="language" value="English" />
+		<info name="usage" value="16K" />
 		<part name="cass1" interface="zx81_cass">
 			<dataarea name="cass" size="9070">
 				<rom name="Sabotage.tzx" size="9070" crc="0120fe03" sha1="4d123b512a757fe2e0bfd7ddee111322d8238d9f"/>
 			</dataarea>
 		</part>
 	</software>
 
+	<software name="sabotage_txp" cloneof="sabotage">
+		<description>Sabotagem (Timex Portugal)</description>
```

Comment:
And again you ignored my previous advice, base the shortname on the actual title so 'sabotagm'.